docker/start-iip-proc.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+PORT=$1
+while true; do
+    if ! lsof -i:$CURRENT_PORT &>/dev/null; then
```

Comment:
`$CURRENT_PORT` seems to be referencing undefined var - I think it probably means to use `$PORT`?

```suggestion
    if ! lsof -i:$PORT &>/dev/null; then
```